src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,126 @@
+package duke;
+
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.Todo;
+import duke.ui.Greetings;
+
+import java.util.Scanner;
+
+public class Duke {
+
+	public static void startDuke() {
```

Comment:
This method is a bit long and has a lot of responsibilities. You might consider create a new methods for each case. For example in case userMessage.startsWith("todo"), you can make a method to validate the command and call it in the case block. So that the startDuke method will be shorter and prettier